operations.go

```diff
@@ -121,6 +136,21 @@ func newOpsType(stack *stackType) *opsType {
 			}
 			return []float64{float64(fact)}, 1, nil
 		}},
+
+		ophandler{"fib", "Calculate fibonacci of x", 1, func(a []float64) ([]float64, int, error) {
```

Comment:
Why not design `fib()` in a way that it can be called directly from the ophandler?